<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PayloadDebugger</class>
 <widget class="QWidget" name="PayloadDebugger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>270</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PayloadDebugger</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>381</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Receive Area</string>
   </property>
   <widget class="QPlainTextEdit" name="rxPayloadEdit">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>20</y>
      <width>370</width>
      <height>90</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>381</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Transmit Area</string>
   </property>
   <widget class="QPlainTextEdit" name="payloadEdit">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>20</y>
      <width>370</width>
      <height>90</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="slaveIDComboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>To Slave #</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
